.github/workflows/sdk-regression.yml

```diff
@@ -34,6 +34,12 @@ jobs:
       - uses: xt0rted/pull-request-comment-branch@v2
         if: ${{ github.event.issue.pull_request }}
         id: comment-branch
+      - name: Validate branch name
+        run: |
+          if [[ ! "${{ steps.comment-branch.outputs.head_ref }}" =~ ^[a-zA-Z0-9_\-]+$ ]]; then
```

Comment:
lets include forward slashes as well, sometimes we use that

Reply:
Added